src/app/pages/home/index.js

```diff
@@ -1,7 +1,7 @@
 import { Page } from '../../components/page';
 import { SectionContainer, SectionContent } from '../../components/section';
 import { AccountCard } from './accountCard';
-import { FreeAddonsSection } from './freeAddonsSection';
+// import { FreeAddonsSection } from './freeAddonsSection';
```

Comment:
I'd just remove this rather than leave it commented. It we bring it back we can find the diff easily enough.